intl/msg_hash_us.h

```diff
@@ -1059,21 +1059,21 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_RDB_ENTRY_TGDB_RATING,
 MSG_HASH(MENU_ENUM_LABEL_VALUE_REBOOT,
       "Reboot")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORDING_CONFIG_DIRECTORY,
-      "Recording Config Dir")
+      "Recording Config (dir)")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORDING_OUTPUT_DIRECTORY,
-      "Recording Output Dir")
+      "Recording Output (dir)")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORDING_SETTINGS,
       "Recording")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_RECORD_CONFIG,
-      "Record Config")
+      "Load Record Config")
```

Comment:
Good point. I caught that too... but that commit was already made hours ago. I can fix that too. I don't want to step on too many toes. We can have another go on everything in the future.